### DISCUSSION TITLE
tower.im网站的icon全部被替换，无法显示

### DISCUSSION THREAD

**Discussion by witt-bit** [F9y4ng/GreasyFork-Scripts#223] [问答专区(Question / Answer)]
替换后的图片

![image](https://github.com/F9y4ng/GreasyFork-Scripts/assets/52407727/aef956ee-5bc4-44ab-bfa0-b5b345fe1da6)

![image](https://github.com/F9y4ng/GreasyFork-Scripts/assets/52407727/130227e0-0b22-4428-a06c-972702a93c9a)

替换前的图片

![image](https://github.com/F9y4ng/GreasyFork-Scripts/assets/52407727/3e1af94d-7c80-4b26-81ef-626b854a4329)

![image](https://github.com/F9y4ng/GreasyFork-Scripts/assets/52407727/69be871a-4406-434b-a50b-8c9cab0a26ec)

**Comment by F9y4ng** [comment]
抱歉，这种又要关注公众号，又要手机号，又要短信验证的请提供测试账号。
